pkg/controller/vitessshard/reconcile_disk.go

```diff
@@ -105,7 +105,9 @@ func (r *ReconcileVitessShard) reconcileDisk(ctx context.Context, vts *planetsca
 
 	// If disk size has changed and the changes are all ready, mark the shard as ready to cascade. Otherwise, skip this.
 	if anythingChanged {
-		rollout.Cascade(vts)
+		if !*onlineFileSystemExpansion {
```

Comment:
We can move this check to the beginning of `reconcileDisk` where it checks `vts.Spec.UpdateStrategy.Type` and just `return resultBuilder.Result()` because I don't think this function provides any value when using `onlineFileSystemExpansion`